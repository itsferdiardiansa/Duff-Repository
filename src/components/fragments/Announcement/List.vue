<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
        :selectableRows="true"
        :pagination="paginationData"
        @onSelectedRowCallback="handleSelectRow"
      >
        <template #link="props">
          <a :href="props.data.link" target="_blank" class="underline"
            >Selengkapnya</a
          >
        </template>

        <template #status="props">
          <Status :data="props.data.status" />
        </template>

        <template #action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, onMounted, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import ActionButton from './ActionButton'
import Status from './Status'

export default {
  components: {
    Table,
    ActionButton,
    Status,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Title',
        accessor: 'title',
        width: '30%',
      },
      { title: 'Content', accessor: 'content' },
      { title: 'Link', accessor: 'link', align: 'center' },
      { title: 'Status', accessor: 'status', width: '9%', align: 'center' },
      { title: 'Expired', accessor: 'expired', align: 'center' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        align: 'center',
      },
    ])

    const getAnnouncement = () => {
      store.dispatch('announcement/fetchData')
    }

    const filteredData = computed(() => (
      store.getters['announcement/getAnnouncement']
    ))

    const paginationData = computed(() => (
      store.getters['announcement/getPagination']
    ))

    const isFetching = computed(() => (
      store.getters['announcement/getFetchStatus']
    ))

    const searchFilter = data => {
      console.log(data, 'List')
    }

    const handleSelectRow = () => {
      // console.log(data)
    }

    onMounted(() => {
      getAnnouncement()
    })

    return {
      isFetching,
      filteredData,
      paginationData,
      tHeaders,
      searchFilter,
      handleSelectRow
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
