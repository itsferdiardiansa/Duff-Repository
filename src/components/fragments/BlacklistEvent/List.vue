<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:blacklist="props">
          <div>
            <h3 class="font-black textr-lg mb-2">{{ props.data.title }}</h3>

            <p>
              blacklist at: <span>{{ props.data.blacklistAt }}</span>
            </p>
            <p>
              reported at: <span>{{ props.data.reportedAt }}</span>
            </p>
          </div>
        </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="warning">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Blacklist Event Name / Desc / TNC',
        accessor: 'blacklist',
        width: '40%',
        align: 'left',
      },
      {
        title: 'Created Date',
        accessor: 'createdDate',
        width: '60%',
        align: 'left',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.blacklistEvent.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
