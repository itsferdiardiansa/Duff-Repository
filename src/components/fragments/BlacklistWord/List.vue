<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:status="props">
          <Badge
            :textBold="true"
            size="sm"
            variant="danger"
            shape="circle"
            :label="props.data.status"
          />
        </template>

        <template v-slot:blacklist_at> Event Name </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="warning">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Badge from '@common/Badge'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
    Badge,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Word',
        accessor: 'path',
        width: '20%',
        align: 'left',
      },
      {
        title: 'Blacklist at',
        accessor: 'blacklist_at',
        width: '60%',
        align: 'left',
      },
      { title: 'Status', accessor: 'status', width: '5%', align: 'center' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.url_redirection.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
