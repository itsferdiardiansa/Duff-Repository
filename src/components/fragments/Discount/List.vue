<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :isLoading="isFetching"
        :rowLoader="2"
      >
        <template v-slot:eventName="{ data }">
          <div>
            <a :href="data.link" class="text-yellow-600 text-base">{{
              data.title
            }}</a>

            <p class="text-xs">
              creator: <span>{{ data.creator }}</span>
            </p>
          </div>
        </template>

        <template v-slot:discount="{ data }">
          <div class="">
            <p>
              Percentage: <span>{{ data.discount.percentage }}</span>
            </p>
            <p>
              Fixed: <span class="font-black">{{ data.discount.fixed }}</span>
            </p>
          </div>
        </template>

        <template v-slot:discountAmount="{ data }">
          <p class="font-black">{{ data.discount.amount }}</p>
        </template>

        <template v-slot:action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import data from '@mock/collections'
import ActionButton from './ActionButton'

export default {
  components: {
    Table,
    ActionButton,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Title',
        accessor: 'title',
        width: '20%',
        align: 'left',
      },
      {
        title: 'Event Name',
        accessor: 'eventName',
        width: '20%',
        align: 'left',
      },
      { title: 'Start Date', accessor: 'startDate', align: 'center' },
      { title: 'Potential GMV', accessor: 'potentialGMV', align: 'center' },
      { title: 'Discount', accessor: 'discount', align: 'center' },
      {
        title: 'Discount',
        accessor: 'discountAmount',
        width: '10%',
        align: 'center',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.discount.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
