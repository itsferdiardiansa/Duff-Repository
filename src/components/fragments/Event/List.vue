<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :emptyDataComponent="emptyDataComponent"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:imgThumb="props">
          <div class="flex justify-center">
            <div class="img-thumb w-24 inline-block">
              <img :src="props.data.imgThumb" />
            </div>
          </div>
        </template>

        <template v-slot:event="props">
          <div>
            <div class="title">
              <h3 class="font-black text-lg">
                {{ props.data.title }}
              </h3>
            </div>

            <div class="flex mt-2">
              <div>
                <p>Total ticket sold: <strong>0</strong></p>
                <p>Total paid: <strong>0</strong></p>
              </div>

              <div class="ml-5">
                <p>
                  Event Date: <strong>{{ props.data.eventDate }}</strong>
                </p>
                <p>
                  Event Type: <strong>{{ props.data.eventType }}</strong>
                </p>
              </div>
            </div>
          </div>
        </template>

        <template v-slot:status="props">
          <div class="text-center">
            <Badge :dot="true" variant="success" />
            <span class="ml-2">{{
              props.data.status.tier ? 'Active' : 'Inactive'
            }}</span>
            <span class="px-2">|</span>
            <span>{{ props.data.status.role }}</span>
          </div>
        </template>

        <template v-slot:action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import EmptyData from './EmptyData'
import ActionButton from './ActionButton'
import Badge from '@common/Badge'
import data from '@mock/collections'

export default {
  components: {
    Table,
    Badge,
    ActionButton,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Image',
        accessor: 'imgThumb',
      },
      {
        title: 'Event',
        accessor: 'event',
      },
      {
        title: 'Created Date',
        accessor: 'created_date',
      },
      {
        title: 'Status',
        accessor: 'status',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '20%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.event.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    const emptyDataComponent = computed(() => EmptyData)

    return {
      isFetching,
      filteredData,
      tHeaders,
      emptyDataComponent,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
