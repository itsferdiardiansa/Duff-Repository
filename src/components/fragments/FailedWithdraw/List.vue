<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :isLoading="isFetching"
        :rowLoader="2"
      >
        <template v-slot:event_detail="props">
          <h3 class="font-black text-lg mb-3">{{ props.data.title }}</h3>

          <p>
            Date: <span class="font-bold">{{ props.data.date }}</span>
          </p>
          <p>
            Time: <span class="font-bold">{{ props.data.time }}</span>
          </p>
        </template>

        <template v-slot:organization="props">
          <div class="flex">
            <div class="img-thumb">
              <img :src="props.data.imgThumb" class="w-20" />
            </div>

            <div class="ml-3">
              <h3 class="font-black text-lg mb-2">
                {{ props.data.organizationName }}
              </h3>

              <p class="text-xs mb-1">{{ props.data.email }}</p>
              <p class="text-xs">{{ props.data.phone }}</p>
            </div>
          </div>
        </template>

        <template v-slot:widthdraw_detail="props">
          <p>
            Requested Date:
            <span class="font-bold">{{ props.data.withdrawDetail.date }}</span>
          </p>
          <p>
            Withdraw Amount:
            <span class="font-bold">{{
              props.data.withdrawDetail.amount
            }}</span>
          </p>
          <p>
            Error Message:
            <span class="font-bold">{{
              props.data.withdrawDetail.errorMsg
            }}</span>
          </p>
        </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="dark">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Event Detail',
        accessor: 'event_detail',
        width: '30%',
        align: 'left',
      },
      { title: 'Organization', accessor: 'organization', align: 'left' },
      { title: 'Withdraw Detail', accessor: 'widthdraw_detail', align: 'left' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.failedWithdraw.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
