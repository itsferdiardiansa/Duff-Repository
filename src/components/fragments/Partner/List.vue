<template>
  <div class="list">
    <Modal
      title="Delete confirmation"
      description="Are you sure you want to delete this item?"
      :onConfirmFn="deleteData"
    />

    <div class="list--header">
      <Button
        label="Create Footer"
        variant="primary"
        :bold="true"
        :icon="['fa', 'plus']"
        @click="createPartner"
      />
    </div>

    <Table
      :headers="tHeaders"
      :items="filteredData"
      :isFetching="requestStatus.fetch"
      :onError="requestStatus.error.status"
      :onFailedFetchHandler="fetchData"
      :pagination="pagination"
      :onPageChange="handlePageChange"
    >
      <template #status="{ data: { is_active } }">
        <div class="status">
          <Badge
            size="xs"
            :bold="true"
            :variant="parseInt(is_active) ? 'success' : 'danger'"
            :dot="true"
            :pill="true"
          />
          <span class="ml-2">{{
            parseInt(is_active) ? 'active' : 'inactive'
          }}</span>
        </div>
      </template>

      <template #logo="{ data }">
        <img :src="data.logo" class="h-10 inline-block" />
      </template>

      <template #order="{ data }">
        <OrderButton :data="data" />
      </template>

      <template #action="{ data }">
        <ActionButton :data="actionButtons" :item="data" />
      </template>
    </Table>
  </div>
</template>
<script>
import { onMounted, computed, ref, reactive } from 'vue';
import { useRouter } from 'vue-router';
import { useStore } from 'vuex';
import Table, { ActionButton } from '@common/Table';
import Badge from '@common/Badge';
import Button from '@common/Button';
import Modal from '@common/Modal';
import EmptyData from './EmptyData';
import OrderButton from './OrderButton';

export default {
  components: {
    Table,
    Badge,
    Button,
    Modal,
    ActionButton,
    OrderButton,
  },
  setup() {
    const store = useStore();
    const router = useRouter();
    const params = reactive({ page: 1, limit: 5 });
    const tHeaders = ref([
      {
        title: 'Partner Name',
        accessor: 'description',
        width: '20%',
        align: 'center',
      },
      { title: 'Icon', accessor: 'logo' },
      { title: 'Tagline', accessor: 'tagline' },
      { title: 'Order', accessor: 'order', width: '10%', align: 'center' },
      { title: 'Status', accessor: 'status', width: '10%', align: 'center' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '25%',
        align: 'center',
      },
    ]);

    const actionButtons = ref([
      {
        icon: ['fa', 'pencil-alt'],
        variant: 'dark',
        onClickFn: (e, data) => {
          router.push({
            name: 'Update Partner',
            params: { data: JSON.stringify(data) },
          });
        },
      },
      {
        icon: ['fa', 'trash'],
        variant: 'dark',
        onClickFn: (e, data) => {
          toggleModal(e, data);
        },
      },
    ]);

    const toggleModal = (e, data) => {
      $modal.show(data);
    };

    const fetchData = () => {
      store.dispatch('partner/fetchData', params);
    };

    const pagination = computed(() => {
      return store.getters['partner/getPagination'];
    });

    const filteredData = computed(() => {
      return store.getters['partner/getItems'];
    });

    const requestStatus = computed(() => {
      return store.getters['partner/getRequestStatus'];
    });

    const emptyDataComponent = computed(() => EmptyData);

    const deleteData = ({ hash_id }) => {
      store.dispatch('partner/deleteData', {
        action: 'form.delete',
        message: 'Succesfully delete',
        status: 'success',
        hash_id,
        params,
      });
    };

    const handlePageChange = pageParams => {
      Object.assign(params, pageParams);

      fetchData();
    };

    const createPartner = () => {
      router.push('/partner/create');
    };

    const getPath = path => '/e/' + path;

    onMounted(fetchData);

    return {
      requestStatus,
      pagination,
      filteredData,
      fetchData,
      tHeaders,
      emptyDataComponent,
      createPartner,
      getPath,
      actionButtons,
      deleteData,
      handlePageChange,
    };
  },
};
</script>
<style lang="scss" scoped>
.list {
  &--header {
    @apply py-6 flex justify-end items-center;
  }

  .status {
    @apply flex items-center justify-center;
  }
}
</style>
