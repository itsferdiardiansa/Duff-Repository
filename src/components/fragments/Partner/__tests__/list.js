import { mount } from '@vue/test-utils';
import List from '@fragment/Partner/List';
import partnerModule from '@store/modules/partner';
import { createStore } from 'vuex';

describe('fragment/Partner/List', () => {
  let store;

  beforeAll(() => {
    store = createStore({
      modules: {
        partner: partnerModule,
      },
    });
  });

  it('will render and match snapshot', async () => {
    const Component = mount(<List />, {
      global: {
        plugins: [store],
      },
    });

    await Component.vm.$store.dispatch('partner/fetchData', {
      page: 1,
      limit: 10,
    });
    expect(Component.html()).toMatchSnapshot();
  });
});
