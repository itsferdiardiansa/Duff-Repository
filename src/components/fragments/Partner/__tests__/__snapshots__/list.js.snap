// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fragment/Partner/List will render and match snapshot 1`] = `
<div class="list">
  <!--v-if-->
  <div class="ss-table-filter">
    <div class="ss-table-filter--top">
      <div class="ss-table-filter--input-form">
        <form class="ss-form search-form">
          <div class="ss-control--input has-icon bg-gray-100"><span class="icon"><svg xmlns="http://www.w3.org/2000/svg" class="ss-form-control--icon" viewBox="0 0 512 512"><path fill="currentColor" d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"></path></svg></span><input class="ss-input bg-gray-100" type="search" placeholder="Search" autocomplete="off">
            <!--v-if-->
            <!--v-if-->
          </div>
        </form>
      </div>
      <div class="ss-table-filter--extend"><button class="ss-button ss-button--orange ss-button--base text-bold">
          <div class="ss-button--wrapper">
            <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 448 512">
                <path fill="currentColor" d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"></path>
              </svg></div>
            <div class="ss-button--label has-icon">Create Partner</div>
          </div>
        </button></div>
    </div>
    <div class="ss-table-filter--bottom">
      <div class="ss-table-filter--category"><label class="ss-table-filter--title">Sort By</label>
        <div class="w-40 ss-control--input w-40">
          <div class="ss-select">
            <div class="ss-select--button" tabindex="0"><span class="button-label">Newest</span><span class="button-icon"><svg xmlns="http://www.w3.org/2000/svg" class="h-4" viewBox="0 0 512 512"><path fill="currentColor" d="M8 256C8 119 119 8 256 8s248 111 248 248-111 248-248 248S8 393 8 256zm231-113.9L103.5 277.6c-9.4 9.4-9.4 24.6 0 33.9l17 17c9.4 9.4 24.6 9.4 33.9 0L256 226.9l101.6 101.6c9.4 9.4 24.6 9.4 33.9 0l17-17c9.4-9.4 9.4-24.6 0-33.9L273 142.1c-9.4-9.4-24.6-9.4-34 0z"></path></svg></span></div>
            <div class="ss-select--dropdown slide-top-enter-from slide-top-enter-active" style="display: none;">
              <ul class="dropdown-list">
                <li class="dropdown-item">Newest</li>
                <li class="dropdown-item">Oldest</li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <table class="ss-table">
    <thead class="ss-table--head" headers="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]" items="[object Object],[object Object],[object Object]" isfetching="false" pagination="[object Object]" emptymessage="Empty data" rownumber="true" rowloader="5" selectablerows="false" withpagination="true" withfilter="true" filteroptions="">
      <tr>
        <th width="4%">
          <!--v-if--><span>#</span>
        </th>
        <th class="" width="20%" align="center">Partner Name</th>
        <th class="">Icon</th>
        <th class="">Tagline</th>
        <th class="" width="10%" align="center">Order</th>
        <th class="" width="10%" align="center">Status</th>
        <th colspan="3" class="" width="25%" align="center"></th>
      </tr>
    </thead>
    <tbody class="ss-table--body">
      <!--v-if-->
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>1</label></td>
        <td class="ss-table--body-col text-center">
          <div>
            <p>Text</p>
          </div>
        </td>
        <td class="ss-table--body-col"><img src="" class="h-10 inline-block"></td>
        <td class="ss-table--body-col">
          <div>This is tagline</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="flex justify-center">
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm disabled" disabled="">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform -rotate-90"></div>
                </div>
              </button></div>
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm" textbold="true">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform rotate-90"></div>
                </div>
              </button></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="status">
            <div class="ss-badge ss-badge--success ss-badge--xs text-bold pill dot">
              <!--v-if-->
            </div><span class="ml-2">active</span>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>2</label></td>
        <td class="ss-table--body-col text-center">
          <div>1</div>
        </td>
        <td class="ss-table--body-col"><img src="https://loket-development.s3.amazonaws.com/30845ae2-7139-4def-bd9c-962ab523ba44" class="h-10 inline-block"></td>
        <td class="ss-table--body-col">
          <div>taglinne</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="flex justify-center">
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm disabled" disabled="">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform -rotate-90"></div>
                </div>
              </button></div>
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm" textbold="true">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform rotate-90"></div>
                </div>
              </button></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="status">
            <div class="ss-badge ss-badge--success ss-badge--xs text-bold pill dot">
              <!--v-if-->
            </div><span class="ml-2">active</span>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>3</label></td>
        <td class="ss-table--body-col text-center">
          <div>Deskripsi partner</div>
        </td>
        <td class="ss-table--body-col"><img src="https://loket-development.s3.amazonaws.com/images/temporary/20210219/1613712872_przeXh.png" class="h-10 inline-block"></td>
        <td class="ss-table--body-col">
          <div>Ini tagline partner</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="flex justify-center">
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm disabled" disabled="">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform -rotate-90"></div>
                </div>
              </button></div>
            <div class="mr-2"><button class="ss-button ss-button--light ss-button--sm" textbold="true">
                <div class="ss-button--wrapper">
                  <!--v-if-->
                  <div class="ss-button--label"><img svg-inline="" class="w-3 transform rotate-90"></div>
                </div>
              </button></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="status">
            <div class="ss-badge ss-badge--success ss-badge--xs text-bold pill dot">
              <!--v-if-->
            </div><span class="ml-2">active</span>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
  <!--v-if-->
</div>
`;
